products/cms-integration/overview.md

```diff
@@ -0,0 +1,94 @@
+
+# Table of Contents
+
+1.  [CMS Export Content Model Transformation Overview](#org452f1e3)
+    1.  [Purpose](#orga07c918)
+    2.  [10,000ft view](#orgaf4354e)
+    3.  [Adding a new content model](#org77f88f4)
+    4.  [Zooming in](#orgacd6399)
+    5.  [Debugging](#org68c9208)
+    6.  [Best practices](#orgc3b70c7)
+
+
+
+<a id="org452f1e3"></a>
+
+# CMS Export Content Model Transformation Overview
+
+That&rsquo;s a mouthful, isn&rsquo;t it?
+
+
+<a id="orga07c918"></a>
+
+## Purpose
+
+To outline how content gets from the CMS to static HTML files.
+
+
+<a id="orgaf4354e"></a>
+
+## 10,000ft view
+
+New content begins its life in the CMS. When it grows up, it gets published.
+That process is out of scope for this document. We start our story after content
+is published.
+
+The Drupal CMS will bundle the CMS export into a tarball and make it available to
+the build script for processing.
+
+When the content build is run, it does a lot of things. The steps that are
+pertinent to this document are:
+
+1.  Pulling the CMS export tarball
+2.  Transforming the CMS export content models to the template
+    content models
+3.  Applying the content models to the templates to generate static HTML
+
+
+<a id="org77f88f4"></a>
+
+## Adding a new content model
+
+When adding a new content model, you&rsquo;ll need to add a handful of new files to
+`src/site/stages/build/process-cms-exports/`:
+
+-   Pre-transformation (raw) schema
```

Comment:
Suggestions: 
- Summarize the process and how each of these files fits into that. -
-  order the files needed according to order that they're used in the transformation process.